pkg/apis/core/v1alpha1/conversions.go

```diff
@@ -400,6 +402,36 @@ func Convert_core_ProjectMember_To_v1alpha1_ProjectMember(in *core.ProjectMember
 	return nil
 }
 
+func Convert_core_KubeletConfig_To_v1alpha1_KubeletConfig(in *core.KubeletConfig, out *KubeletConfig, s conversion.Scope) error {
```

Comment:
😵 Why do we need this conversion? Can't you just use `*resource.Quantity` also in the internal API?

Reply:
yes, thanks